test/pseudo-tty/test-tty-color-support-warning-2.out

```diff
@@ -1,3 +1,3 @@
 
-(node:*) Warning: The 'NODE_DISABLE_COLORS' env is ignored due to the 'FORCE_COLOR' env being set.
-(Use `* --trace-warnings ...` to show where the warning was created)
+[31m(node:*) Warning: The 'NODE_DISABLE_COLORS' env is ignored due to the 'FORCE_COLOR' env being set.
+(Use `* --trace-warnings ...` to show where the warning was created)[0m
```

Comment:
```suggestion
(Use `* --trace-warnings ...` to show where the warning was created)[0m

```

Reply:
I will revert this file. Seems we don't need this anymore.